Urban-Fun/View Controllers/TimelineViewController.m

```diff
@@ -114,6 +115,14 @@ - (IBAction)didTapLogout:(id)sender {
     }];
     
 }
+- (void) didPostActivity:(Activity *)activity{
+    NSMutableArray *temporaryActivitiesArray = [[NSMutableArray alloc] initWithArray:self.arrayOfActivities];
```

Comment:
Is this code working? Remember we don't want to commit code that is not working properly, to me seems like this code is not working fine? Can you explain what is happening in this piece of code?

I think you can watch [this tutorial](https://youtu.be/wKNuzW66Jd0?t=966), seems like this is what you want to accomplish here

Reply:
I removed this code.